annalieseschropp/checkers
# 117 text visibility and end game bug fix

This PR closes #117 and #111 

### What Was Accomplished
* Update to all text elements to increase visibility
* Added overlays to increase visual consistency
* Updated all popups to all be uniform and follow the colour scheme
* Ensured all anchor points work
* Fixed bug regarding end game
* All tests still pass and all features function as expected

### Testing
* To run unit tests, ensure you have a build set for the **all** scenes
    * `File > Build Settings > Add Open Scenes > Build`
* You also need the Test Runner window open to see the available tests
    * `Window > General > Test Runner`
    * You can dock this where you would like based on your layout preferences
* Click `Run All` in the Test Runner window or select a specific test then click `Run Selected`
* To manually run the scene, press the `play` button or go to `File > Build and Run`
* For manual testing
    * Check anchor points
    * Check that visuals all work
    * Check that the bug is fixed
    * Check that it all looks good
    * Let me know if you hate anything lol

### To dos
* If anything looks bad, I'll change it
